app/libraries/config.ts

```diff
@@ -19,4 +19,4 @@ export const configProject = {
   colorTo: 'to-gray-700',
   textColor: 'text-gray-700',
   frameworks: [],
-} satisfies Library
```

Comment:
any reason for this change? it means these don't get the literal types, they get the broader type of `Library` and types like "union of all IDs" can't be built out of them

Reply:
Ah! I couldn't find a downside but that would be one. I did this because there were type errors for things like `library.badge` in the file I was working in. I'll just change it back and leave the type errors.

Reply:
Basically trying to solve this: https://github.com/microsoft/TypeScript/issues/52805

But the ruling is it's supposed to work this way, so optional properties on a type that uses satisfies have to be explicitly set to undefined. I'll do that.